plugin/pkg/admission/persistentvolume/label/admission.go

```diff
@@ -75,6 +77,13 @@ func (l *persistentVolumeLabel) Admit(a admission.Attributes) (err error) {
 		}
 		volumeLabels = labels
 	}
+	if volume.Spec.GCEPersistentDisk != nil {
```

Comment:
@markturansky @saad-ali This should be part of volume plugins, I think.  It should NOT be an open-coded list of cases.  I'll file a new issue.